### PR TITLE
Added management command for generating and deleting fake buildings.

### PR DESCRIPTION
**Pull request by mateesville93**
### How has it been tested?

The new custom command can be called using 
`python manage.py buildings <number> <--argument>`
required arguments:
```
 --delete          
 --create
```
#### Examples:
Generate 10K fake buildings:
`python manage.py buildings 10000 --create`
Delete 10K buildings:
`python manage.py buildings 10000 --delete`
Help command:
`python manage.py buildings --help`

```shell script
cd path/to/repo
docker-compose exec api bash
root@ba4fd81f9023:/code# python manage.py buildings 30 --create
100% |███████████████████████████████| 30/30 [00:00<00:00, 37.89it/s]
Successfully created 30 buildings.
root@ba4fd81f9023:/code# python manage.py buildings 25 --delete
Successfully deleted 25 buildings.
```

### CODE REVIEW

File: api/buildings/management/commands/buildings.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                self.style.ERROR(
+                    "Missing required argument: --create or --delete"
+                )
+            )
+            self.stdout.write(
+                self.style.SUCCESS(
+                    "Try: python manage.py buildings %s --create"
+                    % (int(options["total_buildings"]))
+                )
+            )
```
Comment (tudoramariei) [inline api/buildings/management/commands/buildings.py:106]: Not sure if this is supposed to be here

File: api/buildings/management/commands/buildings.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                self.style.SUCCESS(
+                    "Successfully deleted %s building(s)."
+                    % (int(buildings_count))
+                )
+            )
+        elif options["create"]:
+            for building_id in trange(int(options["total_buildings"])):
+                address = fake.address()
+                risk_category = random.choice(
+                    ["U1", "U2", "U3", "U4", "RS1", "RS2", "RS3", "RS4"]
```
Comment (tudoramariei) [inline api/buildings/management/commands/buildings.py:50]: While the RS1-4 is not the right naming for the risk categories, what we have currently is not right either. Will have to implement `SeismicCategoryChoice` and change the model.

Comment (tudoramariei) [reply api/buildings/management/commands/buildings.py:50]: Also, would add a `"N/A"` option (which is currently found as `"n/a"`

File: api/buildings/management/commands/buildings.py (hunk 1 above)
Comment (mateesville93) [reply api/buildings/management/commands/buildings.py:106]: It's just a suggestion with the correct command.

File: api/buildings/management/commands/buildings.py (hunk 2 above)
Comment (mateesville93) [reply api/buildings/management/commands/buildings.py:50]: Ok I will implement `SeismicCategoryChoice`. Please tell me the right categories!

Comment (tudoramariei) [reply api/buildings/management/commands/buildings.py:50]: The proper categories, in the order of their importance, are the following: 
1. U1
2. U2
3. U3
4. U4
5. RS I
6. RS II
7. RS III
8. RS IV
9. N/A

**Comment by tudoramariei** [comment]
It seems there's still one more thing left here. Would you have the time to change the `risk_category` variables of the fixtures (`api/buildings/fixtures/buildings.json`)? I'm guessing the issue is that it's expecting a maximum 3 character variable and `RS III` or `RS II` are a bit larger than that.
```
django.db.utils.DataError: Problem installing fixture '/code/buildings/fixtures/buildings.json': Could not load buildings.Building(pk=70): value too long for type character varying(3)
```
Link to the failing test: https://github.com/code4romania/seismic-risc/pull/542/checks?check_run_id=1893844052

**Review by tudoramariei — APPROVED** [review]

**Pull request merged**